vipps-login-api.md

```diff
@@ -725,69 +725,99 @@ Web-views should not be used when using Vipps Login in a native application. Ins
+you find more information on how to do this [here](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api-faq.md#how-can-i-activate-and-set-up-vipps-login).
 
-The Vipps app will return some data with the return to the `app_callback_uri`. It contains two query parameters `state` and `resume_uri`. `State` is the OIDC `state` parameter passed at the start of a login which can be used to identify the specific login if needed. 
+The Vipps app will return some data with the return to the `app_callback_uri`. 
+It contains two query parameters `state` and `resume_uri`. 
+`State` is the OIDC `state` parameter passed at the start of a login which can be used to identify the specific login if needed. 
 
-The `resume_uri`parameter is generated by Vipps and can optionally be used to resume the login when the user returns from the Vipps app. If the merchant app manages to keep the initial browser window open it is not required to use the `resume_uri`. To use the `resume_uri` it must be opened in the same browser used in the initial phase (such as SafariViewController or Chrome Custom Tabs). It is required that the browser contains the cookies from the interaction that happens before the user is sent to the Vipps app.
+The `resume_uri`parameter is generated by Vipps 
+and can optionally be used to resume the login when the user returns from the Vipps app. 
+If the merchant app manages to keep the initial browser window open it is not required to use the `resume_uri`. 
+To use the `resume_uri` it must be opened in the same browser used in the initial phase 
+(such as ASWebAuthenticationSession or Chrome Custom Tabs). It is required that the browser contains the cookies 
+from the interaction that happens before the user is sent to the Vipps app.
 
 Example `app_callback_uri` request
 ``` 
 merchant-app://callback/?state=RFiQdrl_lvJUpVmTRSKmsZRGLM0G1N1qh0WebZ1gDNk&resume_uri=https%3A%2F%2Fapi.vipps.no%2Fvipps-login-idp%2Findex.html%3FtabId%3D7607f7f0-7ae2-49b7-9cb5-102143dac4ea
 ```
 
-A typical flow/implementation might look like this:
-1. An OpenID authentication flow authorization URI is generated on the merchant backend. The URI is communicated to the merchant app and is used to initiate Vipps login in an external browser that is opened within the app, see [specification](#using-vipps-login-in-native-applications)for details.
-2. If required, Vipps login will deeplink the user over to the Vipps app (if the user is remembered in the browser the user will be authenticated directly - the user will then be on step 5 below)
-3. When the user has approved the login in the Vipps app the `app_callback_uri` is triggered.
-4. The `app_callback_uri` will return the user to the merchant app. The app now has 2 separate choices.
+#### A typical flow/implementation might look like this:
+![](images/app2appsuggestion.png)
+1. An [OpenID authentication flow authorization URI](#oauth-20-authorize) is generated on the merchant backend. 
```

Comment:
Could we maybe consistently start each sentence with what will perform the action? Might make it clearer who is  controlling each step.
```suggestion
1. Merchant backend generates an [OpenID authentication flow authorization URI](#oauth-20-authorize). 
```